bedrock/firefox/tests/test_helpers.py

```diff
@@ -52,6 +52,7 @@ def check_dumb_button(self, doc):
         eq_(pq(links[5]).attr('href'), settings.APPLE_APPSTORE_FIREFOX_LINK
                                             .replace('/{country}/', '/'))
 
+    @patch('bedrock.base.waffle.switch', Mock(return_value=False))
```

Comment:
Good point about disabled in dev... I think it's fine if it's disabled in most dev envs actually. We could manually enable it for `www-dev.a.o` or any demos, or do like you suggested and default it to off. I still think it should be mostly on in prod mode... Still another option is to avoid the whole `switch` system here and just use a setting we create specifically for this and set it to default to having the sha1 URL on by default. That might be the best thing here, but I'm happy with what we have now really.